src/tribol/geom/GeomUtilities.cpp

```diff
@@ -743,8 +757,8 @@ FaceGeomError Intersection2DPolygon( const real* const RESTRICT xA,
          {
             if (interId >= maxSegInter) 
             {
-               SLIC_DEBUG("Intersection2DPolygon: number of segment/segment intersections exceeds precomputed maximum; " << 
-                          "check for degenerate overlap.");
+              //  SLIC_DEBUG("Intersection2DPolygon: number of segment/segment intersections exceeds precomputed maximum; " << 
```

Comment:
Do we want to guard this with TRIBOL_USE_HOST like is done elsewhere and uncomment it out?